resqpy/olio/wellspec_keywords.py

```diff
@@ -37,91 +37,93 @@
 # NB: changing entries in this list will usually require other code change elsewhere
 # second element of tuple should be >= first element
 # yapf: disable
```

Comment:
Please restore the column aligned, yapf disabled, formatting of these two dictionary tables.